css/filmstrip/_vertical_filmstrip.scss

```diff
@@ -152,11 +152,6 @@
     .filmstrip {
```

Comment:
For some unknown for me reason, if I delete this `.filmstrip-only` then the local thumbnail is added at the bottom of the page 🤷‍♂️

Reply:
Thanks @hristoterezov for the css lesson :) 🤦

Reply:
np :)